mkdocs.yml

```diff
@@ -0,0 +1,10 @@
+site_name: Oauth2 Proxy
+site_author: Philips
+docs_dir: docs
+
```

Comment:
nav:
  - ...
plugins:
  - techdocs-core
  - awesome-pages
  
Should be added if we automatically wnt to generate the doc files